packages/@dcl/sdk/src/setupProxy.ts

```diff
@@ -78,8 +78,6 @@ const setupProxy = (dcl: any, app: express.Application) => {
     }
   }
 
-  app.use(express.json())
```

Comment:
Isn't this needed?

Reply:
i think we are already calling this method at the cli